app-shell/src/labware/definitions.ts

```diff
@@ -75,7 +76,12 @@ export function addLabwareFile(file: string, dir: string): Promise<void> {
 }
 
 export function removeLabwareFile(file: string): Promise<void> {
-  const result = shell.moveItemToTrash(file)
-
-  return result ? Promise.resolve() : fs.unlink(file)
+  return shell.trashItem(file).then(
+    () => {
+      Promise.resolve()
+    },
+    () => {
+      return fs.promises.unlink(file)
+    }
+  )
```

Comment:
Can this be made simpler? This `() => Promise.resolve()` isn't doing anything. Also, since we're using `fs-extra` for now (probably past time we drop it and move back to vanilla `fs`), I don't think we need `fs.promises` unless the typechecker is mad about it

```suggestion
  return shell.trashItem(file).catch(() => fs.unlink(file))
```